src/components/List/Events/List.jsx

```diff
@@ -124,24 +124,17 @@ function Lists(props) {
                   })}
                 </span>
                 <span className="event-list-description-place">
-                  {eventItem?.location?.map((place) => {
-                    if (!place?.isVirtualLocation)
-                      return bilingual({
-                        en: place?.name?.en,
-                        fr: place?.name?.fr,
-                        interfaceLanguage: user?.interfaceLanguage?.toLowerCase(),
-                      });
-                    else if (place?.isVirtualLocation)
-                      return (
-                        <>
-                          {eventItem?.location?.length > 1 && <>&nbsp;|&nbsp;</>}
-                          {bilingual({
-                            en: place?.name?.en,
-                            fr: place?.name?.fr,
-                            interfaceLanguage: user?.interfaceLanguage?.toLowerCase(),
-                          })}
-                        </>
-                      );
+                  {eventItem?.location?.map((place, index) => {
+                    return (
+                      <>
+                        {bilingual({
+                          en: place?.name?.en,
+                          fr: place?.name?.fr,
+                          interfaceLanguage: user?.interfaceLanguage?.toLowerCase(),
+                        })}
+                        {index >= 0 && index <= eventItem?.location?.length - 2 && <>&nbsp;|&nbsp;</>}
```

Comment:
This looks like it will work, but I was thinking simpler like using join().  I made a commit. Take a look and merge if you like it. https://github.com/culturecreates/footlight-app/pull/166/commits/996f87d979fe96605fe68d63707df46c2a450ec9